labonneboite/web/contact_form/views.py

```diff
@@ -50,6 +49,46 @@ def decorated(*args, **kwargs):
     return decorated
 
 
+def is_recruiter_from_lba():
+    return request.args.get('origin', '') == 'labonnealternance'
+
+
+def get_action_name():
+    action_name = request.args.get('action_name', None)
+    if action_name not in ACTION_NAMES:
+        action_name = None
+    return action_name
+
+
+def get_subject():
+    source = "LBA" if is_recruiter_from_lba() else "LBB"
+    return f"Nouveau message entreprise depuis {source}"
+
+
+def generate_fail_flash_content():
+    email = 'labonnealternance@pole-emploi.fr' if is_recruiter_from_lba() else 'labonneboite@pole-emploi.fr'
+    message = """
+            <div class="text-center">
+                Erreur lors de l\'envoi de l\'e-mail.
+                Vous pouvez nous contacter directement à l\'adresse suivante : <a href="mailto:{}">{}</a>
+            </div>
+        """.format(email, email)
+    return Markup(message)
+
+
+def unknown_siret_message():
+    email = 'labonnealternance@pole-emploi.fr' if is_recruiter_from_lba() else 'labonneboite@pole-emploi.fr'
+    msg = """
```

Comment:
DNRY? utiliser `CONTACT_EMAIL` vu dans

```
./roles/deploy-code/files/lbb_production_settings.py:37:CONTACT_EMAIL = 'labonneboite@pole-emploi.fr'
./roles/deploy-code/files/lbb_staging1_settings.py:34:CONTACT_EMAIL = 'labonneboite@pole-emploi.fr'
./roles/deploy-code/files/lbb_staging_settings.py:34:CONTACT_EMAIL = 'labonneboite@pole-emploi.fr'
```

et peut-être faire iso avec l'email LBA

Reply:
Of course, mais ça existait comme ça :)